Model/Service/InvoiceService.php

```diff
@@ -48,19 +17,14 @@ public function __construct(
      */
     public function prepareInvoiceWithoutItems(OrderInterface $order, $amount)
     {
-        try {
-            $invoice = $this->orderConverter->toInvoice($order);
-            $invoice->setBaseGrandTotal($amount);
-            $invoice->setSubtotal($amount);
-            $invoice->setBaseSubtotal($amount);
-            $invoice->setGrandTotal($amount);
+        $invoice = $this->orderConverter->toInvoice($order);
```

Comment:
How does removing the try-catch help here?

Reply:
@matt-thomason it's the bugsnag in catch that's injected via constructor, which we had to override for the purpose. The parent constructor was changed in M2.3.3, by adding one more attribute, so our call to
```php
parent::__construct($repository, $commentRepository, $criteriaBuilder, $filterBuilder, $notifier, $orderRepository, $orderConverter);
```
throws an error. I decided to remove the bugsnag and not override the constructor instead of looking for some more complicated solution that would work with M2.3.3 as well as the older versions. I wrapped up this method call in try / catch + bugsnag, see createOrderInvoice above.